exercises/practice/secret-handshake/SecretHandshakeTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+class SecretHandshakeTest extends PHPUnit\Framework\TestCase
+{
+    private SecretHandshake $secretHandshake;
+
+    public static function setUpBeforeClass(): void
+    {
+        require_once 'SecretHandshake.php';
+    }
+
+    public function setUp(): void
+    {
+        $this->secretHandshake = new SecretHandshake();
+    }
+
+    /**
+     * uuid: b8496fbd-6778-468c-8054-648d03c4bb23
+     */
+    public function testWinkForOne(): void
+    {
+        $this->assertEquals(['wink'], $this->secretHandshake->commands(1));
+    }
+
+    /**
+     * uuid: 83ec6c58-81a9-4fd1-bfaf-0160514fc0e3
+     */
+    public function testDoubleBlinkForTen(): void
+    {
+        $this->assertEquals(['double blink'], $this->secretHandshake->commands(2));
+    }
```

Comment:
I think we could use binary notation here. If students look into the tests to see the actual values passed in, seeing the binary notation helps understanding the task. They will be busy with figuring out the bits in the values when they `echo` or `var_dump()`.

Or maybe mix binary and decimal notation to add some head scratching.

```suggestion
    public function testDoubleBlinkForTen(): void
    {
        $this->assertEquals(['double blink'], $this->secretHandshake->commands(0b0_0010));
    }
```

Reply:
If we change this, the `commands` signature needs to be updated to, currently and `int` is expected. But in general I agree with you, but according to specs it's always `int` values parsed, so the question is should the specs be updated too? If not, I'll keep it as is.

Reply:
Well, `0b0_0000` **is** an int. It is PHPs binary integer notation since at least 2022.

Reply:
Oh.! I wasn't aware of that, then I'm OK with your change. I'll commit suggestion.